### ISSUE TITLE
Register group ID with Sonatype

### ISSUE THREAD

**Issue by nicolaiparlog** [junit-pioneer/junit-pioneer#27]
To release artifacts into Maven Central the group ID must be registered with Sonatype. As per [their documentation](http://central.sonatype.org/pages/choosing-your-coordinates.html) the group ID "reuses the domain name system in a reversed manner", which means there is no choice but to use `org.junit-pioneer`.

**Comment by nicolaiparlog** [comment]
I just [created an issue](https://issues.sonatype.org/browse/OSSRH-28808) to create the ID.

@smoyer64 If you want to be able to release to Maven Central I need your [Sonatype JIRA](https://issues.sonatype.org) account.

**Comment by smoyer64** [comment]
I am smoyer64 there too.

In general, I hope that neither of us is manually releasing projects to Sonatype but I've had instances where things got "stuck" and manual intervention was required.

Thanks!

**Comment by nicolaiparlog** [comment]
[Progress](https://oss.sonatype.org/content/repositories/snapshots/org/junit-pioneer/junit-pioneer/1.0-SNAPSHOT/)! 🎉

**Comment by nicolaiparlog** [comment]
To make sure everything works as planned and to keep this issue open until Sonatype's JIRA issue is closed, this issue is blocked until we actually made a release.

Btw., I just updated Sonatype's issue to include @smoyer64's user name.

**Comment by nicolaiparlog** [comment]
@smoyer64 Sonatype just unlocked you, which means you should be able to publish a snapshot with:

    gradle publish -P snapshotRepoPass=<YOUR_MAVEN_PASS>

Could you give this a try and report back?

**Comment by smoyer64** [comment]
That's not working ... I'm not a gradle expert (yet?) but it looks like I'd be using my password with the username "nipa" with the current configuration.  In Maven I'd set the username and password in my user-level ```~/.m2/settings.xml``` file so it would never become part of my CLI history and I'm wondering if there's a similar facility in Gradle.

**Comment by nicolaiparlog** [comment]
> I'm wondering if there's a similar facility in Gradle.

Yes, apparently [there is](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_properties_and_system_properties). Good cartch, I'll work on that! Created #46.

**Comment by nicolaiparlog** [comment]
@smoyer64 Could you give publishing a snapshot another try?

**Comment by nicolaiparlog** [comment]
> The only comment that I have is that the property names are a bit generic.

Right, I didn't think of that. Let's change that when we have to.
